lib/schemas/archive-format.json

```diff
@@ -20,7 +20,12 @@
           ]
         },
         "strip_exif": { "type": "boolean" },
-
+        "blocks": {
```

Comment:
I added this change to `next-minor` when merging #8263 as 342d4f7fde6f12cbfc9cc2f438936d099548ee3e (so you can rebase and remove this here)